scripts/assets/test_file_exemptions.textproto

```diff
@@ -5,6 +5,9 @@ exempted_file_path: "app/src/main/java/org/oppia/android/app/activity/ActivityIn
 exempted_file_path: "app/src/main/java/org/oppia/android/app/activity/ActivityModule.kt"
 exempted_file_path: "app/src/main/java/org/oppia/android/app/activity/ActivityScope.kt"
 exempted_file_path: "app/src/main/java/org/oppia/android/app/activity/InjectableAppCompatActivity.kt"
+exempted_file_path: "app/src/main/java/org/oppia/android/app/activity/route/ActivityRouter.kt"
+exempted_file_path: "app/src/main/java/org/oppia/android/app/activity/route/ActivityRouterModule.kt"
```

Comment:
These should have test suites added.

Reply:
Can you help me with the starting point for test. Also, I think we can only write test for ActivityRouter.

Reply:
Sorry, I missed this earlier. Here's a reference:

https://github.com/oppia/oppia-android/blob/efc98ca9cb50374a601d10373a1f8c29182c4675/testing/src/test/java/org/oppia/android/testing/logging/SyncStatusTestModuleTest.kt#L44.

Reply:
Thanks @BenHenning, created a Tests similar to the reference test